### PR TITLE
Fixed the blocking issues on those welcome pages

### PR DESCRIPTION
**Pull request by yuhuitech**
Dear developer:

Hello! I am the creator of issue #386, and I have fixed the blocking issues on those welcome pages. I would appreciate it if you could kindly revise my code and give me some advice. Thank you so much for your precious time!!

Here are screenshots after my changes, and as you can see those elements are separated from each other now and the text view is still scrollable:

copy copy

### CODE REVIEW

**Comment by yuhuitech** [comment]
Hello!

I just spotted another similar problem on the "Outline" page. On this page, if I use the same display settings, the logs could not be seen fully because it is not scrollable.

**Steps to reproduce the behavior:**

1. use the largest font and display in the display setting
2. open this app for the first time, and scroll through all of the welcome pages
3. click "I understand" on the pop up panel
4. click the three vertical dots on the top-right of the main page
5. click "start" then click "stop"
6. this error could be seen

**Screenshot**

(The overlap is marked in red)

copy

And I just fixed this issue as well by making it scrollable. Here is the screenshot after my fix, and I would like to commit those changes later.

copy

**Comment by franklinbill** [comment]
I met the same issue🤝.  I strongly agree with your fix  @yuhuitech 👍

**Comment by franklinbill** [comment]
It seems that this issue is a11y. @yuhuitech

**Comment by renyuneyun** [comment]
Thanks very much for the PR!  Sorry for replying late. I was too busy recently.

The PR looks promising. I'll try to review the details as soon as possible :)

File: app/src/main/res/layout/item_activity_log.xml [hunk 1]
```diff
@@ -1,120 +1,129 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android">
-    <FrameLayout
-        android:layout_width="match_parent"
+
+    <ScrollView
+        android:layout_width="wrap_content"
```
Comment (renyuneyun) [inline app/src/main/res/layout/item_activity_log.xml:5]: Why not `match_parent`?

File: app/src/main/res/layout/fragment_welcome_text.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
         android:id="@+id/title"
         style="@style/AppTheme.Intro.PureText.Heading"
         android:layout_width="0dp"
-        android:layout_height="0dp"
+        android:layout_height="wrap_content"
```
Comment (renyuneyun) [inline app/src/main/res/layout/fragment_welcome_text.xml:12]: Why doesn't `0dp` work?

**Comment by renyuneyun** [comment]
Looks good to me. I'll merge it. Thanks!

Actually I'm stilll thinking why the second case happened. Everything from that `CardView` is `android:layout_height=wrap_content`, which I presume should have exactly the right height for that...

In the meanwhile, I left some comments in the code. They are not very influential, but I'm just wondering XD

**Pull request merged**